### PR TITLE
[4.0] Fix style in mobile view

### PR DESCRIPTION
**Pull request by hitesh-coder**
Pull Request for Issue #32797.

### Steps to reproduce
System > Templates > Mail Template

### Summary of Changes
changed position and size of cog icon also added padding to the following elements.

### Actual result BEFORE applying this Pull Request
Before

### Expected result AFTER applying this Pull Request
After

### Documentation Changes Required
none

### CODE REVIEW

**Comment by brianteeman** [comment]
Please update the title with something meaningful

**Comment by PhilETaylor** [comment]
I think this should be taken as two PRs

1) For the placement of the Cog - as I think a lot of people might disagree with this move as its a "designed element" of the template 

2) The layout of the Mail Templates page when viewed on mobile.

**Comment by hitesh-coder** [comment]
@PhilETaylor but the cog icon looks a little out placed as it also overlaps in other mail template. So i think this position should be fine.

**Comment by PhilETaylor** [comment]
The COG issue has been addressed here https://github.com/joomla/joomla-cms/pull/32818 and here https://github.com/joomla/joomla-cms/pull/32819 as well, all three conflict.

**Comment by hitesh-coder** [comment]
so should I remove my changes for cog-icon as someone is working on it?

**Comment by drmenzelit** [comment]
We would like to see the bottom icon changed too to match size:
image
You can do it in this PR (changing the description appropriately) or you can create a new PR.

**Comment by PhilETaylor** [comment]
Visually it might be better to center the inner cog icon? Just a thought...

Screenshot 2021-03-23 at 11 24 03

**Comment by hitesh-coder** [comment]
### COG Icon
COG

### Burger Icon
Burger

@drmenzelit @PhilETaylor  Thanks for the suggestions

**Comment by ceford** [comment]
I am not sure what we are expecting but on my simulated iPhone 6/7/8 iOS 11 the cog is smaller but so near the top and there is no hamburger icon at all.

screen shot 2021-03-28 at 17 56 37<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32810">issues.joomla.org/tracker/joomla-cms/32810</a>.</sub>

**Comment by brianteeman** [comment]
The burger is at the bottom!

**Comment by ceford** [comment]
I thought folk would not want to see a picture of something not there. Here is the whole screen grab that I see:

screen shot 2021-03-28 at 19 02 47<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32810">issues.joomla.org/tracker/joomla-cms/32810</a>.</sub>

**Comment by hitesh-coder** [comment]
Hey @ceford you would not see a burger icon at the bottom on the edit mail template because it is not present there you have to go to system > template > mail template.

Mail template

I am not sure why it is not working on your side. I will check it once again.

**Comment by hitesh-coder** [comment]
Ok, it was an issue on my side, I have not updated the file on git, my bad. Thanks for checking @ceford

**Comment by ceford** [comment]
I have tested this item :white_check_mark: successfully on 6101f52a51a84fdf659f7bff07d28d08e4ce96b1<br /><br />OK: I now see the cog a burger icons fitting perfectly in the top an bottom bars.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32810">issues.joomla.org/tracker/joomla-cms/32810</a>.</sub>

**Comment by meinhoonharsh** [comment]
I have tested this item :red_circle: unsuccessfully on 6101f52a51a84fdf659f7bff07d28d08e4ce96b1<br /><br />Its all Perfect till width=375px, but at width=320px It is having a Space on the Top-Right in Navbar

## Before:
image


## After at width=375px
image

## After at width=320px
image
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32810">issues.joomla.org/tracker/joomla-cms/32810</a>.</sub>

**Comment by hitesh-coder** [comment]
@meinhoonharsh thanks for mentioning. I have fixed it please check.

**Comment by meinhoonharsh** [comment]
I have tested this item :white_check_mark: successfully on 82a0d03973e4039d9993eaf5cda29000c2d5a1db<br /><br />That Blank Space has been is also not present at width=320px

## At width=320px 

image
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32810">issues.joomla.org/tracker/joomla-cms/32810</a>.</sub>

**Comment by meinhoonharsh** [comment]
I have tested this item :white_check_mark: successfully on 82a0d03973e4039d9993eaf5cda29000c2d5a1db<br /><br />The Blank Space has been Removed at width=320px


### Mobile View at width=320px
image
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32810">issues.joomla.org/tracker/joomla-cms/32810</a>.</sub>

**Comment by hitesh-coder** [comment]
hey @meinhoonharsh which blank space are you referring to

File: administrator/templates/atum/scss/blocks/_sidebar.scss [hunk 1]
```diff
@@ -251,7 +251,7 @@
     border-radius: 40px;
 
     [dir="ltr"] & {
-      right: 10px;
+      right: 4px;
```
Comment (infograf768) [inline administrator/templates/atum/scss/blocks/_sidebar.scss:254]: if you modify in LTR, should it not be also modified in RTL?

File: administrator/templates/atum/scss/blocks/_toolbar.scss [hunk 2]
```diff
@@ -159,23 +163,23 @@
     border: 4px solid var(--atum-bg-light);
 
     [dir=ltr] & {
-      right: 10px;
+      right: 4px;
     }
 
     [dir=rtl] & {
       left: 10px;
```
Comment (infograf768) [inline administrator/templates/atum/scss/blocks/_toolbar.scss]: if you modify in LTR, should it not be also modified in RTL?

**Comment by hitesh-coder** [comment]
@meinhoonharsh and @ceford could you please test it once again. Thanks in advance.

**Comment by ceford** [comment]
I have tested this item :white_check_mark: successfully on 7bdc90c4c0c3ae103667bfdf7c0a15fc378b5714<br /><br />OK - also tested at 320 and 300 pixels wide - cog and burger perfectly placed.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32810">issues.joomla.org/tracker/joomla-cms/32810</a>.</sub>

**Comment by meinhoonharsh** [comment]
I have tested this item :white_check_mark: successfully on 7bdc90c4c0c3ae103667bfdf7c0a15fc378b5714<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32810">issues.joomla.org/tracker/joomla-cms/32810</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32810">issues.joomla.org/tracker/joomla-cms/32810</a>.</sub>

**Comment by drmenzelit** [comment]
Thank you very much for your first contribution! Well done

**Pull request merged**